### ISSUE TITLE
Server side rendering fails due to missing sessionid [possibly due to proxy authentication]

### ISSUE THREAD

**Issue by jbergler** [grafana/grafana#2568]
It appears that in my setup the server side rendering of a graph is failing because the invocation of `phantonjs render.js` is missing the `sessionid` parameter.

The failure is logged simply with `url=<url> png=<filename> width=<width> height=<height> cookiename=<cookiename> sessionid=<sessionid> domain=<domain>`

Adding some debugging to render.js shows that the command line args include `sessionid=`
This fails to match both the regular expression for argument parsing and the `!params.sessionid` check

The request I'm making hits nginx with the a cookie `grafana_sess = b287b1add7d8c540`
Nginx proxies the request to grafana adding a header `X-Forwarded-For: my_username`

I have the following in my `custom.ini`

```ini
[users]
allow_sign_up = true

[auth.anonymous]
enabled = true

[auth.basic]
enabled = false

[auth.proxy]
enabled = true
header_name = X-Forwarded-User
header_property = username
auto_sign_up = true
```

**Comment by torkelo** [comment]
thanks for reporting this, fixed in master (and in latest 2.2 pre release builds)

**Comment by jbergler** [comment]
Verified as working in `Version: 2.2.0-pre1, Commit: v2.1.2+139-g960a4f7, Build date: 2015-08-21 07:28:01 -0700 PDT`

Thanks for the quick fix.
